build/documentation/copy-package-dry-run.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Utility for mocking the copying readme files from packages to .docs/en/packages/[package-name]/readme.md.
+ * Usage:run node build/documentation/copy-package-readme.js --dry-run OR
+ *       npm run docs:dry-run
+ */
+
+const path = require("path");
+const fs = require("fs");
+const glob = require("glob");
+
+const rootDir = path.resolve(process.cwd());
+const srcReadmePaths = "packages/*/README.md";
+const destDir = path.join("docs", "en", "packages");
+
+/**
```

Comment:
I actually think it is more maintainable to keep dry-run as part of the actual script implementation. The reason being is that if we ever need to change `copy-package-readme.js`, we would now need to *also* make changes to the dry-run script. Adding this as a distinct script adds the opportunity for the two behaviors to get out of sync.

I think the implementation you had prior to extracting this out was good - just check the dry-run flag prior to writing any file. That provides more assurance as to what the script will actually do.

Reply:
Okay, makes sense. I'll switch it back